src/rv/actions.go

```diff
@@ -78,6 +81,17 @@ func NodeIP(c *cli.Context) {
 	fmt.Fprintln(stdout, ip)
 }
 
+func sortedNodeNames(nodeList NodeList) []string {
```

Comment:
This is actually the supper efficient version of how to do this, in non critical code this could probably be shortened to

```Go
func sortedNodeNames(nodeList NodeList) []string {
	names := []string{}

	for name, _ := range nodeList {
		names = append(names, name)
	}

	sort.Strings(names)
	return names
}
```

The above code is deficient because of the characteristics of append. It doubles the capacity each time it reaches capacity. Slices and append are a bit strange in Go, but give you some pretty tight control of memory growth where needed.

https://play.golang.org/p/ml7hMXv-3C

With that said, might as well go for the gold :+1: